zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -197,6 +197,31 @@ public void testReloadAllNotes() throws IOException {
   }
 
   @Test
+  public void testLoadAllNotes() {
+    Note note;
+    try {
+      assertEquals(0, notebook.getAllNotes().size());
+      note = notebook.createNote(anonymous);
+      Paragraph p1 = note.addParagraph();
+      Map config = p1.getConfig();
+      config.put("enabled", true);
+      p1.setConfig(config);
+      p1.setText("hello world");
+      note.persist(anonymous);
+    } catch (IOException fe) {
+      logger.warn("Failed to create note and paragraph. Safe to return, other tests might be failing as well", fe);
+      return;
+    }
```

Comment:
The error message above implies that it is intended as failing condition for this test, but `return` will actually make test pass.

What was the intention? I think it's fine to explicitly fail the test if some pre-condition fail